tests/msc3030_test.go

```diff
@@ -127,6 +131,38 @@ func TestJumpToDateEndpoint(t *testing.T) {
 				remoteCharlie.JoinRoom(t, roomID, []string{"hs1"})
 				mustCheckEventisReturnedForTime(t, remoteCharlie, roomID, eventB.AfterTimestamp, "b", eventB.EventID)
 			})
+
+			t.Run("when looking backwards before the room was created, should be able to find event that was imported", func(t *testing.T) {
+				t.Parallel()
+				timeBeforeRoomCreation := time.Now()
+				roomID, _, _ := createTestRoom(t, alice)
+
+				// Join from the application service bridge user
+				as.JoinRoom(t, roomID, []string{"hs1"})
+
+				// Import a message in the room before the room was created. We have to
+				// use an application service user because they are the only ones
```

Comment:
Re-commented so I can apply

```suggestion
				// use an application service user because they are the only one
```